client/my-sites/upgrades/index.js

```diff
@@ -252,6 +252,11 @@ module.exports = function() {
 		);
 
 		page(
+			'/checkout/no-site',
```

Comment:
I'm afraid we can't, because of the next route:
```
/checkout/:domain/:product?
```

It'll be the same is the route:
```
/checkout/:cart_key
```

Given `/checkout/tug.blog` we can't determine if that's a cart key or a domain